CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -108,6 +109,12 @@ - (void)viewDidLoad {
     _tableView.estimatedSectionHeaderHeight = 100.0;
     _tableView.sectionHeaderHeight = UITableViewAutomaticDimension;
     
+    _noDataLabel = [[UILabel alloc] init];
```

Comment:
This line should be the following for it to display correctly in the center of the view:

`UILabel *noDataLabel         = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, _tableView.bounds.size.width, _tableView.bounds.size.height)];`